omod/src/main/webapp/resources/htmlforms/PatientReferral.xml

```diff
@@ -182,6 +222,20 @@
                 });
             });
 
+            function getPatientTransferEncounterTypes(encounterType) {
+                var enctype="";
```

Comment:
@dbaluku use the full name of the variable - encounterType

Reply:
Done